parser-typechecker/src/Unison/Server/CodebaseServer.hs

```diff
@@ -1,34 +1,33 @@
-{- ORMOLU_DISABLE -} -- Remove this when the file is ready to be auto-formatted
-{-# LANGUAGE DeriveGeneric #-}
-{-# LANGUAGE DeriveAnyClass #-}
-{-# LANGUAGE TypeFamilies #-}
 {-# LANGUAGE DataKinds #-}
+{-# LANGUAGE DeriveAnyClass #-}
```

Comment:
Most changes in this file is simply formatting after I removed `ORMOLU_DISABLE`.